sqlmesh/core/engine_adapter/base.py

```diff
@@ -59,6 +60,12 @@ class EngineAdapter:
     DEFAULT_BATCH_SIZE = 10000
     DEFAULT_SQL_GEN_KWARGS: t.Dict[str, str | bool | int] = {}
     ESCAPE_JSON = False
+    STRUCT_DIFFER_PROPERTIES = {
```

Comment:
Shall we just instantiate the schema differ in place here instead of maintaining a dict? Its instance is not stateful, isn't it?

Reply:
I went back and forth on that. I don't think it matters too much in the end. This provides a bit more flexibility so I just went with it. If you feel strongly though I can change.

Reply:
i'd prefer iaroslav's suggestion